src/js/tracker.js

```diff
@@ -1433,6 +1484,24 @@
 		return {
 
 			/**
+			 * Get the page view ID as generated or provided by mutSnowplowState.pageViewId.
+			 *
+			 * @return string Page view ID
+			 */
+			getPageViewId: function () {
+				return getPageViewId();
+			},
+
+			/**
+			 * Get the cookie name as cookieNamePrefix + basename + . + domain.
+			 *
+			 * @return string Cookie name
+			 */
+			getCookieName: function(basename) {
+				return getSnowplowCookieName(basename);
+			},
+
+			/**
```

Comment:
I don't know if should just make those public instead of having a public wrapper.